src/mscorlib/corefx/System/Globalization/DateTimeFormatInfo.cs

```diff
@@ -2797,6 +2789,7 @@ void InsertHash(TokenHashValue[] hashTable, String str, TokenType tokenType, int
                                     }
                                 }
                                 // The token to be inserted is already in the table.  Skip it.
+                                return;
```

Comment:
Was this just a bug before?  Are we losing anything by exiting early here, or before we were just doing pointless work?